Makefile

```diff
@@ -23,20 +23,27 @@ USER=root
 GROUP=root
 ifeq ($(OS), centos)
 PKG_INSTALL=yum install -y
-PYTHON_DEPS=rh-python36-setuptools python36-click tox python36-configobj
+PY_VER=36
+PYTHON_DEPS=python${PY_VER}-setuptools python${PY_VER}-click tox python${PY_VER}-configobj
```

Comment:
This line appears to be changing `rh-python${PY_VER}-setuptools` to `python${PY_VER}-setuptools` for CentOS - have you tested that?

Reply:
Hi @smithfarm yes I tested on the OS in PR description: "make install tested on Ubuntu, CentOS, OpenSUSE 15, Archlinux".  I was checking why package had non-standard name and the other package name works and is more standard.